base/src/main/java/info/metadude/kotlin/library/c3media/models/RelatedEvent.kt

```diff
@@ -0,0 +1,15 @@
+package info.metadude.kotlin.library.c3media.models
+
+import com.squareup.moshi.Json
+
+data class RelatedEvent (
```

Comment:
Please reformat the file here. The space between the class name and the bracket should be removed.